api/api.go

```diff
@@ -0,0 +1,3 @@
+package api
```

Comment:
I'd say that it's better to put DTO structures into separate package from start.
E.g. to api/types or api/dto package. But it's not mandatory. At least for now.

Reply:
Moved as prompted.